packages/lab/src/__tests__/__e2e__/drawer/Drawer.cy.tsx

```diff
@@ -7,12 +7,12 @@ const { Default, Top, Right, Bottom } = composedStories;
 
 describe("GIVEN a Drawer", () => {
   describe("WHEN no props are provided", () => {
-    it("THEN it should display a scrim by default", () => {
+    it("THEN it should display an overlay by default", () => {
       cy.mount(<Default />);
 
       cy.findByRole("button", { name: /Open Drawer/i }).click();
 
-      cy.get(".saltScrim").should("be.visible");
+      cy.get(".saltDrawer-overlay").should("be.visible");
```

Comment:
Is there a way to check for a role instead?

Reply:
there's no role on the overlay, we only add `role="dialog"` to the actual Drawer. Is there an additional role we should be using here?

Reply:
Sorry I meant just in general, checking this for overlay is fine :)